app/assets/javascripts/initializers/initializeRuntime.js

```diff
@@ -0,0 +1,5 @@
+/* global Runtime */
+
+function initializeRuntime() {
+  document.body.dataset.runtime = Runtime.currentContext();
+}
```

Comment:
This is a tiny, one line initializer. I added it in a separate initializer file because it felt valuable to be separate concern, but looking back to it I'm not sure if that's true.

The alternative would be to simply add this one line to `app/assets/javascripts/initializePage.js`. I'm open to moving it there if others feel like that would be better.